### PR TITLE
Sync ID with Layer 140

### PR DESCRIPTION
**Pull request by Rolariali**
Syncronization ID with layer 140

### CODE REVIEW

File: src/TLMessage/TLMessage/ClientMessages/get_all_chats.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
  */
 class get_all_chats implements TLClientMessage
 {
-    private const CONSTRUCTOR = -341307408; // 0xeba80ff0
+    private const CONSTRUCTOR = 2271179966;
```
Comment (Ririconj) [inline src/TLMessage/TLMessage/ClientMessages/get_all_chats.php:17]: Structure changed likewise

File: src/Registration/RegistrationFromTgApp.php [hunk 2]
```diff
@@ -248,7 +248,6 @@ private function performLoginWorkFlow(callable $cb): void
             new get_pinned_dialogs(),
             new get_state(),
             new get_app_config(),
-            new get_blocked_contacts(),
```
Comment (Ririconj) [inline src/Registration/RegistrationFromTgApp.php:251]: Remove unused class?

File: src/TLMessage/TLMessage/ClientMessages/send_message.php [hunk 3]
```diff
@@ -12,7 +12,7 @@
  */
 class send_message implements TLClientMessage
 {
-    private const CONSTRUCTOR = 1376532592; // 0x520c3870
+    private const CONSTRUCTOR = 228423076;
```
Comment (Ririconj) [inline src/TLMessage/TLMessage/ClientMessages/send_message.php:15]: Structure changed likewise

File: src/Registration/RegistrationFromTgApp.php (hunk 2 above)
Comment (Rolariali) [reply src/Registration/RegistrationFromTgApp.php:251]: Yes
New telegram version don't send the request

File: src/TLMessage/TLMessage/ClientMessages/send_message.php (hunk 3 above)
Comment (Rolariali) [reply src/TLMessage/TLMessage/ClientMessages/send_message.php:15]: Yes
Add just the optional fields

**Pull request merged**